styles/styles.css

```diff
@@ -338,3 +337,68 @@ main .section[data-section-status='initialized'] {
 main .section.highlight {
   background-color: var(--highlight-background-color);
 }
+
+body > main > div> div {
```

Comment:
i would be more specific here and apply to `section`

Reply:
done